Library/Formula/clang-format.rb

```diff
@@ -1,18 +1,18 @@
 class ClangFormat < Formula
   desc "Formatting tools for C, C++, Obj-C, Java, JavaScript, TypeScript"
   homepage "http://clang.llvm.org/docs/ClangFormat.html"
-  version "2015-09-01"
+  version "3.7.1"
```

Comment:
Something like `version "2016-3.7.1"` might work, if you're keen to ditch as much of the date as possible. You can tweak it locally and then run `brew outdated` to find what would be upgraded and what wouldn't.